lib/datasource.js

```diff
@@ -1586,11 +1594,11 @@ DataSource.prototype.discoverSchemas = function(tableName, options, cb) {
     // Default name mapper
     nameMapper = function mapName(type, name) {
       if (type === 'table' || type === 'model') {
-        return fromDBName(name, false);
+        return pascalCase(name);
```

Comment:
@agnes512 A question about the behaviour change:

Before we allowed camel case for table name, and IIUC this PR converts all the table names to be pascal case, there is no option to config it to be other naming conventions including camel case. 
Is it intended?

Reply:
I don't think we allowed table/model names to be camelCase before because every time we checked the type:

```ts
if (type === 'table' || type === 'model') {
      return fromDBName(name, false); // this is saying not to use camelCase, use pascal instead
```
the `false` in the second param was set, no options were allowed to configure it.

And since it was converting _all_ table/model names to pascalCase, I replace it with a formal function `pascalCase()` to make it more legit.

Before the function `fromDBName` handled cases change for type `table`, `model`, `fk`, and `column`. ~I change it to just handle column names only.~ I change it to just handle **column names** and **fk** only
( I realized that I made a mistake here, I should let it handle `fk` as well as it's part of property names).

Another reason I think table and model names should just be pascal case is because lots of our naming conventions are based on the **model** name. We assume model names will be Pascal case, and file names that are generated  based on the model names will be snake case, etc. I think allow customized table/model might cause some unexpected issues.